### DISCUSSION TITLE
ReferenceError: Cannot access 'store' before initialization

### DISCUSSION THREAD

**Discussion by midfar** [vuejs/pinia#1632] [Help and Questions]
### Reproduction

https://vkceyugu.cdn.bspapp.com/VKCEYUGU-fba5454d-3a27-4ea5-babf-ea74eb4644f5/649066cc-3178-4393-8064-b59df4403c29.zip

### Steps to reproduce the bug

1. npm install
2. npm run dev:test
3. open browser and console, login, username=admin, password=123456
4. auto redirect to dashboard, open the file: src/views/dashboard/index.vue
5. goto line 26, modify console.log('dashboard created 123456') to trigger hot reload
6. the console error: ReferenceError: Cannot access 'store' before initialization

### Expected behavior

no error, can hot reload

### Actual behavior

ReferenceError: Cannot access 'store' before initialization
[hmr] Failed to reload /src/views/dashboard/index.vue. This could be due to syntax errors or importing non-existent modules. (see errors above)

**Comment by crazynds** [comment]
I have the same problem
